src/core/server/http/router/validator/validator.ts

```diff
@@ -238,7 +238,7 @@ export class RouteValidator<P = {}, Q = {}, B = {}> {
       // if options.body.output === 'stream'
       return schema.stream();
     } else {
-      return schema.object({}, { allowUnknowns: true });
+      return schema.nullable(schema.object({}, { allowUnknowns: true }));
```

Comment:
__question:__ is there any chance we can use `schema.record()` instead of `schema.object({}, { allowUnknowns: true })`? Technically they're the same, just `record` looks a bit cleaner. Feel free to ignore if it's not that easy as I think.